src/fasta.rs

```diff
@@ -1171,11 +1171,15 @@ mod tests {
     }
 
     #[test]
-    fn test_dna_invalid_dna_whitespace() {
-        assert_parse_err!(
+    fn test_dna_dna_whitespace() {
+        assert_parse!(
             ">Virus1\nAAAA \n",
             FastaParser::<DnaSequence>::strict(),
-            FastaParseError::ParseError(TranslationError::BadNucleotide(' '))
+            vec![FastaRecord {
+                header: "Virus1".to_string(),
+                contents: "AAAA".parse().unwrap(),
+                line_range: (1, 3),
+            }]
         );
     }
```

Comment:
I think this test should be moved, it was grouped with the `*_invalid_*` tests but now it's testing that the parse is successful.